docker/ci/config/windows-setup.ps1

```diff
@@ -11,7 +11,12 @@
 [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
 
 # Install Scoop as Administrator User here
-iex "& {$(irm get.scoop.sh)} -RunAsAdmin"
+# Scoop version >= 0.5.0 has error with Select-CurrentVersion function
+# They have not fix the issue and will error out on Windows ltsc2019
+# https://github.com/ScoopInstaller/Scoop/issues/6180
+# A temp solution is to hardcode the scoop version to 0.4.2
+# iex "& {$(irm get.scoop.sh)} -RunAsAdmin"
+iex "& {$(irm https://raw.githubusercontent.com/peterzhuamazon/scoop-Install/refs/heads/stable/install.ps1)} -RunAsAdmin"
```

Comment:
We can check this is this build repo right ?

Reply:
We can, but this script is currently forked from scoopinstaller/install directly and it is easier to consume their upstream changes without have to copy paste every time in build repo.

Just a temp solution until they fix upstream entirely.

Reply:
I mean this script https://raw.githubusercontent.com/peterzhuamazon/scoop-Install/refs/heads/stable/install.ps1 peter.

Reply:
Yes, and that script is easier to consume upstream changes without have to update build repo every time upstream changes.